plugin.video.carnet-meduza/resources/lib/modules/control.py

```diff
@@ -0,0 +1,235 @@
+# -*- coding: utf-8 -*-
+
+'''
+    Exodus Add-on

```

Comment:
This is a module from a banned add-on. I'm not sure what needs to be done to sanitize for inclusion in the kodi.org repo